deployment/allyabase_setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Dependencies:
+# git
+# nodejs
+# npm
+
+set -e
+
+tmpDir="${TMPDIR:-${XDG_RUNTIME_DIR:-/tmp}}"
+buildDir="${1:-$tmpDir}"
+[[ $buildDir != /* ]]&& buildDir="${PWD%/}/$buildDir"
+buildDir="${buildDir#./}/allyabase"
+ecosystem_config='ecosystem.config.js'
+services=(
+    'julia'
+    'continuebee'
+    'joan'
+    'pref'
+    'bdo'
+    'fount'
+    'addie'
+    'aretha'
+    'sanora-dot-club'
+)
+
+setup_services() {
+    mkdir "$buildDir"&& { cd "$buildDir"|| :; }
+
+    for service in "${services[@]}"; do
+        git clone "https://github.com/planet-nine-app/$service"
+
+        printf '%s\n' "Installing '$service'..."
+        npm install "$service/src/server/node"
+    done
+} # setup_services
+
+setup_ecosystem() {
+    npm install pm2-runtime
+
+    printf '%s\n' \
+        'module.exports = {' \
+        '  apps: [' >>"$ecosystem_config"
+
+    for service in "${services[@]}"; do
+        if [[ $service == 'addie' ]]; then
+            env="{
+                LOCALHOST: 'true',
+                STRIPE_KEY: '<api key here>',
+                STRIPE_PUBLISHING_KEY: '<publishing key here>',
+                SQUARE_KEY: '<api key here>'
```

Comment:
Would it make more sense to have the deployers of this update the keys here, or pass them in as env vars and/or args?